app/src/main/res/layout/sync_tabs_list_item.xml

```diff
@@ -8,56 +8,50 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:paddingStart="60dp"
-    android:paddingEnd="20dp"
-    android:minHeight="@dimen/library_item_height"
     android:background="?android:attr/selectableItemBackground">
 
     <TextView
         android:id="@+id/synced_tab_item_title"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        android:layout_marginStart="12dp"
-        android:ellipsize="end"
+        android:layout_marginStart="72dp"
```

Comment:
Since we use these values multiple times they would be good candidates for pulling into `dimens.xml`

Reply:
Should I just pull values that used multiple times or all of them?